tests/erlang_tests/test_ets.erl

```diff
@@ -386,3 +387,16 @@ test_update_counter() ->
     10 = ets:update_counter(Tid, patatas, {3, 10, 10, 0}),
     0 = ets:update_counter(Tid, patatas, {3, 10, 10, 0}),
     ok.
+
+
+test_delete_table() ->
+    Tid = ets:new(test_delete_table, []),
+    true = ets:insert(Tid, {foo, tapas}),
+    [{foo, tapas}] = ets:lookup(Tid, foo),
+    true = ets:delete(Tid),
+    ok = expect_failure(
+        fun() -> ets:insert(Tid, {gnu, gnat}) end
+    ),
+    Ntid = ets:new(test_delete_table, []),
+    true = ets:delete(Ntid),
```

Comment:
We should also test what happens both when deleting something that has been already deleted, and when deleting twice the same thing.

Reply:
We also don't test for deleting non-existent table. Additionally, I don't get why we're doing insert + lookup if we don't do anything with it.